PAO-Project/src/com/company/Main.java

```diff
@@ -0,0 +1,12 @@
+package com.company;
```

Comment:
[Fix] iti trebuie un .gitignore in care sa adaugi folderul de .idea si fisierul .iml generat de intellij, cat si folderul /out, cu clasele compilate de java